firefox-ios/firefox-ios-tests/Tests/XCUITests/IntegrationTests.swift

```diff
@@ -59,11 +59,12 @@ class IntegrationTests: BaseTestCase {
             app.navigationBars[AccessibilityIdentifiers.Settings.FirefoxAccount.fxaNavigationBar],
             timeout: TIMEOUT_LONG
         )
-        mozWaitForElementToExist(app.staticTexts["Continue to your Mozilla account"], timeout: TIMEOUT_LONG)
```

Comment:
I could've add the invisible space unicode character in `staticTexts`, but the line could become hard to debug.